packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart

```diff
@@ -67,12 +70,38 @@ const _queryParamsKey = 'queryParams';
 const _keyKey = 'key';
 const _valueKey = 'value';
 
-const iosCheckNameToDomainError = <String, DomainError>{
+const _fileFormatKey = 'FILE_FORMAT';
+final iosCheckNameToDomainError = <String, DomainError>{
   'EXISTENCE': IosDomainError.existence,
   'APP_IDENTIFIER': IosDomainError.appIdentifier,
   'HTTPS_ACCESSIBILITY': IosDomainError.httpsAccessibility,
   'NON_REDIRECT': IosDomainError.nonRedirect,
-  'FILE_FORMAT': IosDomainError.fileFormat,
+};
+const aasaFileFormatSubCheck = {
+  'APPLINKS_FORMAT': AASAfileFormatSubCheck.appLinksFormat,
+  'APPLINKS_SUBSTITUTION_VARIABLES_FORMAT':
+      AASAfileFormatSubCheck.appLinksSubstitutionVariablesFormat,
+  'DEFAULTS_FORMAT': AASAfileFormatSubCheck.defaultsFormat,
+  'DEFAULTS_PERCENT_ENCODED_FORMAT':
+      AASAfileFormatSubCheck.defaultsPercentEncodedFormat,
+  'DETAIL_FORMAT': AASAfileFormatSubCheck.detailsFormat,
+  'DETAIL_APP_ID_FORMAT': AASAfileFormatSubCheck.detailsAppIdFormat,
+  'DETAIL_PATHS_FORMAT ': AASAfileFormatSubCheck.detailsPathsFormat,
+  'DETAIL_DEFAULTS_FORMAT': AASAfileFormatSubCheck.detailsDefaultsFormat,
+  'DETAIL_DEFAULTS_PERCENT_ENCODED_FORMAT':
+      AASAfileFormatSubCheck.detailsDefaultsPercentEncodedFormat,
+  'DETAIL_DEFAULTS_CASE_SENSITIVE_FORMAT ':
+      AASAfileFormatSubCheck.detailsDefaultsCaseSensitiveFormat,
```

Comment:
nit: use DETAILS instead of DETAIL to match plurality of "details" in the map values

Reply:
I agree DETAILS instead of DETAIL here is better but the strings here like DETAIL_DEFAULTS_PERCENT_ENCODED_FORMAT are copied from ads validation API in [universal_link_domain_validation.proto] and I have to follow them.